packages/wonder-blocks-form/src/components/text-area.tsx

```diff
@@ -312,45 +317,45 @@ const styles = StyleSheet.create({
         }px`,
     },
     default: {
-        background: color.white,
-        border: `1px solid ${color.offBlack50}`,
-        color: color.offBlack,
+        background: semanticColor.surface.primary,
+        border: `${border.width.hairline}px solid ${semanticColor.border.strong}`,
+        color: semanticColor.text.primary,
         "::placeholder": {
-            color: color.offBlack64,
+            color: semanticColor.text.secondary,
```

Comment:
Same comment here around if we want to have a separate token for placeholder text color! (comment from other PR: https://github.com/Khan/wonder-blocks/pull/2438#discussion_r1927547218)

Reply:
Discussed with Design and they are good with this approach.